docker-compose.yml

```diff
@@ -106,3 +108,7 @@ services:
       dockerfile: ./dev.spark.Dockerfile
     depends_on:
       - server_spark
+networks:
```

Comment:
You don't need to modify the docker-compose.yaml

Reply:
Removed